internal/resources/addons/coredns.go

```diff
@@ -244,6 +255,9 @@ func (c *CoreDNS) decodeManifests(ctx context.Context, tcp *kamajiv1alpha1.Tenan
 func (c *CoreDNS) mutateClusterRoleBinding(ctx context.Context, tenantClient client.Client) (controllerutil.OperationResult, error) {
 	crb := &rbacv1.ClusterRoleBinding{}
 	crb.SetName(c.clusterRoleBinding.GetName())
+	labels := crb.GetLabels()
+	labels[constants.ProjectNameLabelKey] = constants.ProjectNameLabelValue
+	crb.SetLabels(labels)
```

Comment:
This should be done for all the CoreDNS objects, AFAIK:

```go
c.serviceAccount, c.clusterRoleBinding, c.clusterRole, c.service, c.configMap, c.deployment
```

Reply:
Done!